Lib/test/test_complex.py

```diff
@@ -553,6 +594,8 @@ def test_hash(self):
             x /= 3.0    # now check against floating point
             self.assertEqual(hash(x), hash(complex(x, 0.)))
 
+        self.assertEqual(hash(2000005 - 1j), -2)
```

Comment:
```suggestion
        self.assertNotEqual(hash(2000005 - 1j), -1)
```
?

Reply:
That could work. It's still not great in that it's testing an implementation detail rather than testing the API, but it's less likely to be broken by future changes.